### PR TITLE
AIP for secp256r1 for transaction authentication

### PR DESCRIPTION
**Pull request by mstraka100**

### CODE REVIEW

File: aips/secp256r1-ecdsa-for-transaction-authentication.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Interface
+created: 10/18/2023
+updated (*optional): <mm/dd/yyyy>
+---
+
+# AIP-X - Secp256r1 Ecdsa for Transaction Authentication
+
+Passkeys are a novel form of authentication utilizing signing with cryptographic private keys in trusted execution envirionments, built on the WebAuthn standard, meant to replace passwords.
```
Comment (alinush) [inline aips/secp256r1-ecdsa-for-transaction-authentication.md:15]: Typo: envirionments

Comment (mstraka100) [reply aips/secp256r1-ecdsa-for-transaction-authentication.md]: fixed

File: aips/secp256r1-ecdsa-for-transaction-authentication.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Interface
+created: 10/18/2023
+updated (*optional): <mm/dd/yyyy>
+---
+
+# AIP-X - Secp256r1 Ecdsa for Transaction Authentication
+
+Passkeys are a novel form of authentication utilizing signing with cryptographic private keys in trusted execution environments, built on the WebAuthn standard, meant to replace passwords.
```
Comment (alinush) [inline aips/secp256r1-ecdsa-for-transaction-authentication.md:15]: Add link to WebAuthn standard:

File: aips/secp256r1-ecdsa-for-transaction-authentication.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+However, the WebAuthn standard currently supports a limited range of signing algorithms and curves, including ECDSA over `secp256r1`.
+This AIP introduces support for `secp256r1` ECDSA as a transaction authenticator for Aptos.
+
+## Summary
+
+In Aptos, each transaction contains a transaction authenticator that includes a signature and a public key, while the transaction itself contains the sender of the transaction. To verify that a transaction is properly signed, the verifier validates that the public key verifies the signature across the transaction and that the hash of the public key is stored on-chain in a hashed form under the account. By completing this verification, the verifier can be certain that the owner of the account indeed authorizes this transaction. This AIP adds support for `secp256r1` ECDSA for transaction authentication.
+
+## Motivation
+
+Passkeys built on the WebAuthn standard supports `secp256r1`. The following OS platforms support ECDSA over `secp256r1`:
```
Comment (alinush) [inline aips/secp256r1-ecdsa-for-transaction-authentication.md:27]: "supports" -> "support"

**Review by alinush — APPROVED** [review]
The only missing thing is Move support for verifying these signatures, which will be needed for key rotation purposes. Same problem in [AIP-49](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-49.md) cc @davidiw.

**Pull request merged**